README.md

```diff
@@ -210,6 +210,39 @@ Command: <code>python3 -m pip install --user --upgrade pip</code>
 	Link to Solution: <a href="https://github.com/Udayraj123/OMRChecker/issues/70#issuecomment-1268094136">#70</a>
 </details>
 
+### Running in a Container Runtime (Docker or Podman)
+
+The repo contains a containerfile that can be used to run ORM checker without 
+interfering with a system python installation.
+The following instructions are based on using `docker` but can be adapted to
+`podman` by exchanging the names of the two programs.
+
+
+The container can be built by:
+
+1. Cloning the repository.
+```bash
+git clone https://github.com/Udayraj123/OMRChecker
+cd OMRChecker/
+```
+1. Building the container image.
+```bash
+docker build -f containerfile -t ormchecker:python3.11-slim
+```
+1. Creating input and output directories.
```

Comment:
Let's not have this dependency, we can create a small bash file to accept command line arguments during `docker run`. The default will be `inputs` but user can pass a custom directory thru bash args if they want.